packages/usdk/packages/upstreet-agent/packages/react-agents/classes/conversation-object.ts

```diff
@@ -19,6 +20,7 @@ import { loadMessagesFromDatabase } from '../util/loadMessagesFromDatabase';
 
 export class ConversationObject extends EventTarget {
   agent: ActiveAgentObject; // the current agent
```

Comment:
Thinking about the data model more: do we need the conversation to be aware of the current agent at all? Can the conversation just be a bag of messages and players, while the current agent is tracked external to it?

Reply:
yeah i agree, the conversation doesnt have to include the active agent object, the conversation should only require the Agent represented as a player in the conversation